### PR TITLE
Release 4.8.2

### PR DESCRIPTION
**Pull request by johnhwhite**
- Updated styles for AG Grid 24 with modern theme. [#121](https://github.com/blackbaud/skyux-ag-grid/pull/121)

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.8.2](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.8.2)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.8.2](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.8.2)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.8.2 (2021-07-08)
+
+- Updated styles for AG Grid 24 with modern theme. [#121](https://github.com/blackbaud/skyux-ag-grid/pull/121)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Updated styles for AG Grid version 24 in modern theme. [#121](https://github.com/blackbaud/skyux-ag-grid/pull/121)
```
According to the changelog, we just added support for version 25 at the beginning of June. So why are updating styles for version 24 only if we also support version 25?

Comment (johnhwhite) [reply CHANGELOG.md]: Should have been "version 24 and above."

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.8.2](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.8.2)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**